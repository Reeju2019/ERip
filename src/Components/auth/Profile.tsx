import React from 'react'

const Profile: React.FunctionComponent = () => {
  return (
    <div>
      <h1>{window.localStorage.getItem('phone')}</h1>
    </div>
  )
}

export default Profile
