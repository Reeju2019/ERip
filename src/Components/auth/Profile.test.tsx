import { render, screen } from '@testing-library/react'
import Profile from './Profile'
import renderer from 'react-test-renderer'

test('auth profile', () => {
  render(<Profile />)
})

describe('App rendering specification', () => {
  it('SnapShot testing', () => {
    const component = renderer.create(<Profile />)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
