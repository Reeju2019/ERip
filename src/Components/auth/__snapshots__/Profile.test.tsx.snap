// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App rendering specification SnapShot testing 1`] = `
<div>
  <h1 />
</div>
`;
